plugin/matchmaker.vim

```diff
@@ -67,7 +68,8 @@ function! s:default_needle()
     if mode() == 'v'
         return '\V\<'.escape(s:get_visual_selection(), '\').'\>'
     else
-        if getline(".")[col(".")-1] !~# '[[:punct:][:blank:]]' 
+        "Decho 'current char under cursor: '. getline(".")[col(".")-1]
+        if getline(".")[col(".")-1] =~# '[[:alnum:]]' 
```

Comment:
@idning Can you use `\k` here instead of `[[:alnum:]]` ? This would be consistent with `<cword>`, which is based on the value of `'iskeyword'`, which `\k` matches.